JumpscaleLibs/sal/zosv2/ZOSv2.py

```diff
@@ -78,22 +93,79 @@ def reservation_register(self, reservation, expiration_date, identity=None, expi
         reservation.customer_signature = me.nacl.sign_hex(reservation.json.encode())
 
         resp = self._actor_workloads.workload_manager.reservation_register(reservation)
+        if j.core.myenv.config.get("DEPLOYER") and customer_tid:
```

Comment:
What is exactly this doing here ?
I guess this is so it is easier for the user keep a trace of what he as reserved ?
I wonder if this really belongs in the SAL. I feels like we are mixing purpose here.
Also where is this model queried from ? I didn't find a place where the data stored here is actually use.

Reply:
it was mainly to keep track of the people that created reservations, so that later on when we add the feature of use existing network we can filter out if the reservation they are using is actually created by them. If we don't keep track now then we wont have this information later on. Its supposedly going to be replaced later on when the feature is added in a future release. We can add it elsewhere, but we thought it should be added in the sal to make sure any reservation created passes by the check

Reply:
it's for future when we want to reuse resources created from a central solution.